.github/workflows/release.yml

```diff
@@ -2,8 +2,14 @@ name: Release
 
 on: workflow_dispatch
 
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
```

Comment:
Is the env variable `GH_TOKEN` available to all jobs, i.e. is it also passed to the reusable workflow `release_container_image`?

Reply:
Yes, secrets in github are always available to all jobs if defined in